pages/linux/modinfo.md

```diff
@@ -0,0 +1,11 @@
+# modinfo
+
+> Extract information about a Linux kernel module.
+
+- List all attributes of a kernel module:
+
+`modinfo {{kernel_module}}`
+
+- List value(s) of the specified field(s) instead of all the fields:
```

Comment:
You mention "attributes" above and "fields" in this example. Please use any one to be consistent. 
 
Also, no need to say "instead of all the fields". The first example clarifies it that not setting any flags shows all attributes.

Reply:
Updated it further. Please check if it is appropriate now.